src/Makefile.gtest.include

```diff
@@ -69,7 +69,10 @@ pastel_gtest_SOURCES +=\
 	gtest/test_txid.cpp\
 	gtest/test_upgrades.cpp\
 	gtest/test_validation.cpp\
-	gtest/test_zip32.cpp
+	gtest/test_zip32.cpp\
```

Comment:
try to order files by name - this way it's much easier to find smth

Reply:
fixed.Thanks